charter/index.html

```diff
@@ -208,6 +211,12 @@ <h3 id="out-of-scope">Out of Scope</h3>
             <p>The following features are out of scope, and will not be addressed by this Working group.</p>
 
           <ul class="out-of-scope">
+            <li>
+              Definition of a unique and exclusive solution for Personal Data Stores
+            </li>
+            <li>
+              Definition or use of technologies departing from <a href="https://www.w3.org/TR/webarch/">Web Architecture</a>
+            </li>
```

Comment:
Why would this need to be said?

```suggestion
```

It is kind of a given.

Coordination with TAG is expected (and as mentioned).

I also don't see any of the deliverables moving forward in the Recommendation track in particular when a W3C Decision is required if there is anything along the lines of the bullet point here. Hence, it is a "given" to anyone familiar with the Process or willing to abide by it.

Reply:
Yes, you are right